code/soa/intelligence/README.md

```diff
@@ -31,9 +54,21 @@ https://github.com/M-Chase/Maze
 
 https://github.com/M-Chase/Sql_work
 
-#### 汇编语言实验课设
```

Comment:
please list below M-chase and number as 2
:)

Reply:
the code should be positioned in dir /evalcsu/docs/faulty/soa/intelligence/README.md